src/Symfony/Component/Form/Resources/translations/validators.fa.xlf

```diff
@@ -4,15 +4,135 @@
         <body>
             <trans-unit id="28">
                 <source>This form should not contain extra fields.</source>
-                <target>این فرم نباید شامل فیلد اضافه ای باشد.</target>
+                <target>این فرم نباید شامل فیلد اضافه‌ای باشد.</target>
             </trans-unit>
             <trans-unit id="29">
                 <source>The uploaded file was too large. Please try to upload a smaller file.</source>
-                <target>فایل بارگذاری شده بسیار بزرگ می باشد. لطفا فایل کوچکتری را بارگذاری نمایید.</target>
+                <target>فایل بارگذاری‌شده بسیار بزرگ است. لطفاً فایل کوچک‌تری را بارگذاری نمایید.</target>
             </trans-unit>
             <trans-unit id="30">
                 <source>The CSRF token is invalid. Please try to resubmit the form.</source>
-                <target>توکن CSRF نامعتبر می باشد. لطفا فرم را مجددا ارسال نمایید.</target>
+                <target>توکن CSRF نامعتبر است. لطفاً فرم را مجدداً ارسال نمایید.</target>
+            </trans-unit>
+            <trans-unit id="99">
+                <source>This value is not a valid HTML5 color.</source>
+                <target>این مقدار یک رنگ معتبر HTML  نیست.</target>
+            </trans-unit>
+            <trans-unit id="100">
+                <source>Please enter a valid birthdate.</source>
+                <target>لطفاً یک تاریخ تولد معتبر وارد نمایید.</target>
+            </trans-unit>
+            <trans-unit id="101">
+                <source>The selected choice is invalid.</source>
+                <target>گزینه‌ی انتخاب‌شده نامعتبر است</target>
+            </trans-unit>
+            <trans-unit id="102">
+                <source>The collection is invalid.</source>
+                <target>مجموعه‌ی انتخاب‌شده نامعتبر است.</target>
```

Comment:
Note that the user didnt select anything here. 

So it is more "the collection is invalid" rather than "the selected collection is invalid.

Reply:
you are right.